### PR TITLE
New package: youtube-dl.youtube-dl.nightly version 2024.08.07

### LINKED ISSUE #171049: [Package Request]: youtube-dl nightly
### How can we help?

None

### Please read and ensure the following

- [X] The installer meets the above requirements

### Please provide the following information

Download Page Url: https://github.com/ytdl-org/ytdl-nightly
Publisher: ytdl-nightly 
Package Name: youtube-dl
Description: Nightly builds for youtube-dl. 
Package Version: 2024.08.07
Installer URL: https://github.com/ytdl-org/ytdl-nightly/releases/download/2024.08.07/youtube-dl.exe

### PR DESCRIPTION
**Pull request by Dragon1573**
## Checklist for Pull Requests

- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Resolve #171049

## Manifests

- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/171068)

### CODE REVIEW

File: manifests/y/youtube-dl/youtube-dl/nightly/2024.08.07/youtube-dl.youtube-dl.nightly.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Created with komac v2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: youtube-dl.youtube-dl.nightly
+PackageVersion: 2024.08.07
+PackageLocale: en-US
+Publisher: youtube-dl
+PublisherUrl: https://youtube-dl.org/
```
Comment (Dragon1573) [inline manifests/y/youtube-dl/youtube-dl/nightly/2024.08.07/youtube-dl.youtube-dl.nightly.locale.en-US.yaml:8]:
```suggestion
PublisherUrl: https://github.com/ytdl-org
```

> [!WARNING]
>
> Due to a [ruling](https://openjur.de/u/2466945.html) of the Hamburg Regional Court, access to both https://yt-dl.org/ and https://youtube-dl.org/ is blocked.

**Comment by Dragon1573** [comment]
Verified in _Windows 11 `amd64` Sandbox_ instance.

image

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> Executable C:\Users\Validator\AppData\Local\Microsoft\WinGet\Packages\youtube-dl.youtube-dl.nightly__DefaultSource\youtube-dl.exe returned exit code: 2
> Executable C:\Users\Validator\AppData\Local\Microsoft\WinGet\Links\youtube-dl.exe returned exit code: 2

(Automated response - build 894.)

**Comment by stephengillie** [comment]
Verify:
image

**Review by stephengillie — APPROVED** [review]

**Pull request merged**